lambdas/transcode/index.py

```diff
@@ -75,16 +85,30 @@ def lambda_handler(request):
     except ValueError:
         return make_json_response(400, {'error': "Invalid 'duration'"})
 
+    format_params = []
+
+    if format in AUDIO_FORMATS:
+        format_params.extend([
+            '-b:a', '128k',  # 128KB audio bitrate
```

Comment:
Should the requester be able to override this with a query param?

Reply:
Sure.

Though there's also a question: do we care about audio bitrate for videos? And video bitrate?

Reply:
> do we care about audio bitrate for videos?
No for simplicity

> And video bitrate?
Yes if it's simple to parameterize, otherwise No for simplicity